house.php

```diff
@@ -3,7 +3,7 @@
 	znote_visitor_insert_detailed_data(3);
 }
 
-$house = getValue($_GET['id']);
+$house = (isset($_GET['id'])) ? getValue($_GET['id']) : false;
```

Comment:
If Id should be a number or false, we might as well skip string/text sanitizing and just cast it into an integer. 

```suggestion
$house = (isset($_GET['id']) && (int)$_GET['id'] > 0) ? (int)$_GET['id'] : false;
```